# Goal
float32 curr_batt_level

---
# Result
bool battery_charged

---
# Feedback
string charging_battery